### ISSUE TITLE
Wrong text displayed in "get-storage-account" (CLI issue) 

### ISSUE THREAD

**Issue by robbestad** [GenesysGo/shdw-drive-bug-reports#9]
command: `% BROWSER="" shdw-drive get-storage-account -kp XXX` 

```
This is beta software running on Solana's Mainnet. Use at your own discretion.
? Which storage account do you want to unmark for deletion? › - Use arrow-keys. Return to submit.
❯   websites - CEjTaaFFpWymDgBmW2gCL4JNX9v16yjLtp5bohStpskP - 493.5 MiB remaining
```

Note the *Which storage account do you want to unmark for deletion?* part 

Running shdw-drive v0.1.1

**Comment by tracy-codes** [comment]
@robbestad Appreciate you reporting this. Will follow up when this is resolved!

**Comment by tracy-codes** [comment]
This is now resolved in CLI version `0.1.2`. https://www.npmjs.com/package/@shadow-drive/cli/v/0.1.2
